src/components/shared/LaTeX.tsx

```
@@ -0,0 +1,42 @@
+}) => {
+  try {
+    const html = katex.renderToString(children, {
+      displayMode,
+      throwOnError: false,
+      strict: false,
+    });
+
+    return (
+      <span
+        className={`latex-math ${displayMode ? "latex-display" : "latex-inline"} ${className}`}
+        dangerouslySetInnerHTML={{ __html: html }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)